tests/std/tests/P2442R1_views_chunk/test.cpp

```diff
@@ -197,23 +197,30 @@ constexpr bool test_one(Rng&& rng, Expected&& expected) {
         }
     }
 
-    STATIC_ASSERT(CanMemberEnd<const R> == forward_range<const V>);
+    STATIC_ASSERT(CanMemberEnd<const R> == input_range<const V>);
     if constexpr (CanMemberEnd<const R>) {
-        const same_as<sentinel_t<const R>> auto cs = as_const(r).end();
-        assert((r.begin() == cs) == is_empty);
-        STATIC_ASSERT(
-            common_range<R> == (common_range<const V> && (sized_range<const V> || !bidirectional_range<const V>) ));
-        if constexpr (common_range<const R> && bidirectional_range<V>) {
-            if (!is_empty) {
-                assert(equal(*prev(cs), *prev(end(expected))));
-            }
-
-            if constexpr (copy_constructible<V>) {
-                const auto r2 = r;
+        if constexpr (CanMemberBegin<const R>) {
+            const same_as<sentinel_t<const R>> auto cs = as_const(r).end();
+            assert((r.begin() == cs) == is_empty);
+            // clang-format off
+            STATIC_ASSERT(common_range<const R>
+                == (forward_range<const V> && common_range<const V> &&

```

Comment:
Nitpick: We conventionally put binary operators at the beginning of a line, where they are more easily seen. (clang-format would do this automatically, but it's suppressed here.)

It should be possible to deal with suboptimal clang-formatting here without entirely suppressing it. Adding empty comments like `//` will force line breaks.